### ISSUE TITLE
Widgets integration doesn't work

### ISSUE THREAD

**Issue by bullishgopher** [Uniswap/widgets#632]
**Bug Description**
I tried to install widgets on my fresh new nextjs application but it didn't work. Screenshot is attached below

**Steps to Reproduce**

1. Install a fresh new `nextjs` application
2. Install `@uniswap/widgets` and react-redux as described in docs
3. Add **<SwapWidget />** component
4. Got the error

**Expected Behavior**

**Additional Context**

<img width="1586" alt="image" src="https://github.com/Uniswap/widgets/assets/49041460/4cd80878-f842-4f47-840a-e06d1576e0eb">

**Comment by arunkumar201** [comment]
add
"browser": {
    "fs": false
  }
  in package.json 
  this issue will be fixed but later on you might end up of getting same #631 issue

**Comment by bullishgopher** [comment]
Thanks @arunkumar201 

After fixing "fs" error, I am having another issue:

```
Error: Cannot find module '/Users/development/uniswap_dex/nextjs/with-tailwindcss-app/node_modules/.pnpm/@uniswap+widgets@2.59.0_@babel+runtime@7.24.5_@ethersproject+abi@5.7.0_@ethersproject+bignumb_6jidksm52dmfxa3dckdqei34ru/node_modules/@uniswap/conedison/dist/provider/signing'
```

<img width="1573" alt="image" src="https://github.com/Uniswap/widgets/assets/49041460/8b3a9e72-572f-4b8c-859e-ff99056fc0ad">

**Comment by arunkumar201** [comment]
follow -
https://github.com/arunkumar201/uniswap-widget-nextjs

**Comment by arunkumar201** [comment]
No

**Comment by arunkumar201** [comment]
I've utilized a fork of the Uniswap widget, although it may not receive updates in the future.

**Comment by bullishgopher** [comment]
> Thanks @arunkumar201
> 
> After fixing "fs" error, I am having another issue:
> 
> ```
> Error: Cannot find module '/Users/development/uniswap_dex/nextjs/with-tailwindcss-app/node_modules/.pnpm/@uniswap+widgets@2.59.0_@babel+runtime@7.24.5_@ethersproject+abi@5.7.0_@ethersproject+bignumb_6jidksm52dmfxa3dckdqei34ru/node_modules/@uniswap/conedison/dist/provider/signing'
> ```
> 
> <img alt="image" width="1573" src="https://private-user-images.githubusercontent.com/49041460/330509964-8b3a9e72-572f-4b8c-859e-ff99056fc0ad.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU3NDEwMjgsIm5iZiI6MTcxNTc0MDcyOCwicGF0aCI6Ii80OTA0MTQ2MC8zMzA1MDk5NjQtOGIzYTllNzItNTcyZi00YjhjLTg1OWUtZmY5OTA1NmZjMGFkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE1VDAyMzg0OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWMzYTc5YzNhZTZlNzI4OGY1YjI0NjI4N2NjMzMwYjNiNDVlMTU3ZDYxMmUxN2RmNTc2OWJmMjM5NTgyNDQ0MzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.BkuIPyqJuTn5XKvwCERLODs8d_Z1Ji7hmT5WMBITItY">

I forked the widgets that you used but still getting the same error as above

**Comment by bullishgopher** [comment]
> I've utilized a fork of the Uniswap widget, although it may not receive updates in the future.

It worked. Thank you @arunkumar201

**Comment by btc8436954** [comment]
closes #632
